plugins/modules/ovirt_disk.py

```diff
@@ -859,6 +859,7 @@ def main():
                     wait_condition=lambda d: d.status == otypes.DiskStatus.OK,
                     storage_domain=otypes.StorageDomain(name=module.params['image_provider']),
                 )
+
```

Comment:
Do we need that change?

Reply:
This is just a testing change so the CI would fail.